frontend/components/MainPage/header/AgentButton/AgentNotRunningButton.tsx

```diff
@@ -137,21 +130,12 @@ const useServiceDeployment = () => {
     // If was evicted, but can re-stake - unlock the button
     if (isAgentEvicted && isEligibleForStaking) return true;
 
-    // threshold check
-    const masterThresholds =
-      selectedAgentConfig.operatingThresholds[WalletOwnerType.Master];
-    const tokenSymbol =
-      CHAIN_CONFIG[selectedAgentConfig.evmHomeChainId].nativeToken.symbol;
-
-    const safeThreshold = masterThresholds[WalletType.Safe][tokenSymbol];
-
     // SERVICE IS STAKED, AND STARTING AGAIN
     if (isServiceStaked) {
-      const hasEnoughOlas = serviceSafeOlasWithStaked >= requiredStakedOlas;
+      if (!canStartAgent) return false;
```

Comment:
can you please check the above:

```
if (service && isInitialFunded && isServiceStaked) {
      return (serviceTotalStakedOlas ?? 0) >= requiredStakedOlas;
    }
```

I don't think we ever end up here, unless the isInitialFunded is set to false (due to an error probably)